lib/middleware/blp-session.ts

```diff
@@ -21,7 +18,7 @@ var LOGGER: bunyan.Logger = bunyan.createLogger(conf.get('loggerOptions'));
 function createSession(sessionId: string): Promise<blpapi.Session>
 {
     if (!(sessionId in SESSION_STORE)) {
-        var s = new blpapi.Session(DEFAULT_SESSION_OPTIONS);
+        var s = new blpapi.Session(conf.get('sessionOptions'));
```

Comment:
Since the `sessionOptions` configuration doesn't change at runtime, let's cache this as an initialization constant under the `// CONSTANTS` section.